common/system/symbiosis-skel.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Service for symbiosis-skel. Activated by symbiosis-skel.path
+
+[Service]
+ExecStart=/usr/bin/symbiosis-skel 
+User=admin
+Group=admin
```

Comment:
This needs to run as root, rather than admin.  Domains can be owned by anyone,

Reply:
Oh, okay!